hwmgr-plugins/metal3/controller/helpers.go

```diff
@@ -424,7 +424,15 @@ func handleInProgressUpdate(ctx context.Context,
 		if err != nil || tolerate {
 			return hwmgrutils.RequeueWithMediumInterval(), true, err
 		}
+
 		logger.InfoContext(ctx, "BMH update failed", slog.String("BMH", bmh.Name))
+
+		// Clean up the config-in-progress annotation
+		removeConfigAnnotation(node)
+		if err := ctlrutils.CreateK8sCR(ctx, c, node, nil, ctlrutils.PATCH); err != nil {
+			logger.ErrorContext(ctx, "failed to clear config annotation", slog.String("node", node.Name), slog.String("error", err.Error()))
```

Comment:
We don't care if this change is successful here?

Reply:
We do care if this change is successful here.

Reply:
My previous comment was meant to agree with the feedback and acknowledge the issue. I will push an update to address it.

Reply:
Resolved.